src/assets/thirdPartyServices/snet/sentiment_analysis/index.js

```diff
@@ -162,7 +198,9 @@ export default class NamedEntityRecognitionService extends React.Component {
         <Grid item xs={12}>
           <br />
           <br />
-          <FormControl style={{ minWidth: "100%" }}>
+          {
+            /* TODO: Check for the need */
+            /* <FormControl style={{ minWidth: "100%" }}>
```

Comment:
If there is only one service within this custom UI component and the select box is empty, we can delete this code.

Reply:
I think an easy way to verify this would be to check the current dapp for this service.